client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -17,130 +17,277 @@
 
 package org.apache.spark.shuffle;
 
-import java.util.Set;
-
 import com.google.common.collect.Sets;
+import org.apache.spark.internal.config.ConfigBuilder;
+import org.apache.spark.internal.config.ConfigEntry;
 
-import org.apache.uniffle.client.util.RssClientConfig;
+import java.util.Set;
 
 public class RssSparkConfig {
 
+  public static final class SparkConfigBuilder {
+    public static ConfigBuilder configBuilder;
+
+    public static ConfigBuilder key(String key) {
+      configBuilder = new ConfigBuilder(key);
+      return configBuilder;
+    }
+  }
+
+  public static class RssConfigEntry<T> {
+    public ConfigEntry<Object> entry;
+    public String key;
+
+    public RssConfigEntry(ConfigEntry<Object> entry) {
+      this.entry = entry;
+      this.key = entry.key();
+    }
+
+    public T getValue() {
+      return (T) entry.defaultValue(). get();
+    }
+  }
+
+  public static RssConfigEntry<Integer> RSS_PARTITION_NUM_PER_RANGE = new RssConfigEntry(
+      SparkConfigBuilder
+          .key("spark.rss.partitionNum.per.range")
```

Comment:
```
SPARK_PREFIX +  RssClientConfig.RSS_PARTITION_NUM_PER_RANGE
```
?

Reply:
This property `SPARK_PREFIX` is retained and does not affect the use of the concatenated parameters.

Reply:
Could we use here
```
"spark.rss.partitionNum.per.range" -> SPARK_PREFIX +  RssClientConfig.RSS_PARTITION_NUM_PER_RANGE
```
Because Mapreduce also have similar parameter, we want that they can share the common String constant.

Reply:
Remind.

Reply:
done.